android/capacitor/src/main/java/com/getcapacitor/CapConfig.java

```diff
@@ -279,6 +284,14 @@ public boolean isInitialFocus() {
         return initialFocus;
     }
 
+    public int getMinWebViewVersion() {
+        if (minWebViewVersion < MINIMUM_ANDROID_WEBVIEW_VERSION) {
+            return MINIMUM_ANDROID_WEBVIEW_VERSION;
```

Comment:
add a logger warning about the version being smaller than the allowed one and defaulting to MINIMUM_ANDROID_WEBVIEW_VERSION